src/jet/elements/environment.rs

```diff
@@ -141,6 +141,11 @@ impl ElementsEnv {
 impl ElementsEnv {
     /// Return a dummy Elements environment
     pub fn dummy() -> Self {
+        Self::dummy_with(0, elements::Sequence::MAX)
+    }
+
+    /// Return a dummy Elements environment with given locktime
+    pub fn dummy_with(locktime: u32, sequence: elements::Sequence) -> Self {
```

Comment:
In 3adb9a528224e6fe5d1a88ca3c1d93f541acbd4e:

Why not just take a `PackedLockTime` directly instead of a `u32`?